packages/sanity/src/core/form/inputs/ReferenceInput/ReferenceStrengthMismatchAlertStrip.tsx

```diff
@@ -0,0 +1,59 @@
+import {Button, Stack, Text} from '@sanity/ui'
+import {AlertStrip} from '../../components/AlertStrip'
+import {Translate, useTranslation} from '../../../i18n'
+
+/**
+ * Alert strip that shows an explanation and action prompting the user to fix a mismatch in
+ * reference strength, eg when the schema declares it should be weak, but the reference is actually
+ * strong, or wise versa.
+ *
+ * @internal
+ */
+export function ReferenceStrengthMismatchAlertStrip({
+  actualStrength,
+  handleFixStrengthMismatch,
+}: {
+  actualStrength: 'weak' | 'strong'
+  handleFixStrengthMismatch: () => void
```

Comment:
not a blocking comment but should this be `onFixStrengthMismatch`?

Reply:
Lint rules says otherwise actually